packages/nextra/src/loader.ts

```diff
@@ -172,10 +175,8 @@ async function loader(
     : ''
 
   // Imported as a normal component, no need to add the layout.
-  if (!pageImport) {
-    return `${cssImport}
-${result}
-export default MDXContent`
+  if (!isPageImport) {
+    return result
```

Comment:
we don't need import styles for imported markdowns since it will be available from file which imports it